tests/network/autofs_test_nfs_server.pm

```diff
@@ -0,0 +1,32 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Maintainer: Antonio Caristia <acaristia@suse.com>
```

Comment:
Missing summary

Reply:
added summary in `autofs_client` and `autofs_server`